### DISCUSSION TITLE
Optimize the node i/o performance

### DISCUSSION THREAD

**Discussion by in-eth** [cosmos/community#55] [Chain Builders (Cosmos SDK)] [answered]
Hi, I have a problem about cosmos-sdk node. Current node i/o amount is too big. How can I reduce it?

**Comment by tac0turtle** [comment, marked as answer]
reducing iavl cache can help. We are working on reducing overall IO with the help of store v2 and iavl v2. until then reducing the cache and/or upgrading to iavlv1 is best

**Reply by in-eth** [reply]
thanks. will try it
